README.md

```diff
@@ -226,6 +226,15 @@ index them run:
 make reindex
 ```
 
+### 2.6 Cancelling all ongoing (re)indexing operations
+
+
+1) Go to the Simple Queue Service dashboard in the AWS Console. Then, find your target notify SQS queue (should be named azul-notify-...). 
+2) Go to the Lambda dashboard in the AWS Console. Find the azul-indexer...-write lambda. Then, disable the event binding document queue (usually named azul-documents-...). This is done by clicking on the `SQS` trigger in the `Designer` box, clicking on the Enabled switch of azul-documents-... in the newly appeared `SQS` box, then finally saving your settings.
```

Comment:
```suggestion
2) Go to the Lambda dashboard in the AWS Console. Find the `azul-indexer...-write` lambda. Then, disable the event binding to the document queue (usually named `azul-documents-...`). This is done by clicking on the `SQS` trigger in the *Designer* box, clicking on the *Enabled* switch of `azul-documents-...` in the newly appeared *SQS* box, then finally saving your settings.
```

Use italics for labels in user interfaces, things that never change. Use back ticks for values displayed in user interfaces. It's a fuzzy definition, but differentiates text that Amazon chose from text that we chose.